wolfssl/internal.h

```diff
@@ -3607,6 +3607,9 @@ struct WOLFSSL {
 #ifdef WOLFSSL_TLS13
     byte            clientSecret[SECRET_LEN];
     byte            serverSecret[SECRET_LEN];
+#endif
+#ifdef SINGLE_THREADED
+    byte            ownSuites;
```

Comment:
Please move `ownSuites` into `ssl->options` as a bit... `word16 ownSuites:1`. Then it will keep WOLFSSL object size smaller.

Reply:
Done.